lib/elements/autocomplete.js

```diff
@@ -80,15 +79,9 @@ class AutocompletePrompt extends Prompt {
 
     if (this.completing !== p) return;
     this.suggestions = suggestions
-        .map((s, i, arr) => ({ title: getTitle(arr, i), value: getVal(arr, i), description: s.description }))
-        .reduce((arr, sug) => {
-          if (arr[arr.length - 1].length < this.limit)
-            arr[arr.length - 1].push(sug);
-          else arr.push([sug]);
-          return arr;
-        }, [[]]);
+      .map((s, i, arr) => ({ title: getTitle(arr, i), value: getVal(arr, i), description: s.description }));
     this.completing = false;
-    if (!this.suggestions[this.page])
+    if (!this.suggestions)
```

Comment:
nit: probably this `if` statement can go altogether, since it is setting `this.page` which isn't used anymore (or I'm pretty sure it's not based on your diffs).

Reply:
thanks. I'll do a sweep for `this.page` and clean them out. There are a few things that can also be removed like the TODO about skipping to a given page.